src/app/attendance/actions/update-attendance.ts

```diff
@@ -0,0 +1,26 @@
+"use server";
+
+import { Tables } from "@/lib/db";
+import { getSupabaseClient } from "@/utils/supabase/supabaseClient";
+import { AttendanceFormValues } from "../components/AttendanceFormBuilder";
+import { redirect } from "next/navigation";
+import { Routes } from "@/lib/routes";
+
+interface Attendance extends AttendanceFormValues {
+  id: string 
+}
+
+export const updateAttendance = async (attendance: Attendance) => {
+  const supabase = await getSupabaseClient();
+
+  const { error } = await supabase
```

Comment:
Can we add a check here to make sure we only update if attendance hasn't been approved yet?

Reply:
Fixed